src_new/adapters/prebid.js

```diff
@@ -661,86 +821,283 @@ function fetchBids(activeSlots, impressionID){
+			does custom keys do not get used in send-all-bids?
+			do we always need to update the prebid targeting keys config in?
+			what keys in prebid can be re-used?
+	*/
+	return [
+    	//todo: what abt hb_deal, hb_uuid(video?), hb_cache_id(video?), hb_cache_host(video?) ?
+        {
+            key: "pwtpid", //hb_bidder
+            val: function(bidResponse) {
+                return bidResponse.bidderCode;
+            }
+        }, {
+            key: "pwtsid", //hb_adid
+            val: function(bidResponse) {
+                return bidResponse.adId;
+            }
+        }, {
+            key: "pwtecp", //hb_pb
+            val: function(bidResponse) {
+                // return bidResponse.pbMg;
+                return (bidResponse.cpm||0).toFixed(CONSTANTS.COMMON.BID_PRECISION);
+            }
+        }, {
+            key: 'pwtsz', //hb_size
+            val: function (bidResponse) {
+                return bidResponse.size;
+            }
+        }, {
+            key: 'hb_source', //hb_source // we do not want it, so send empty, suppressEmptyKeys feature will prevent it being passed
+            // do not change it in prebid.js project constants file
+            val: function (bidResponse) {
+                // return bidResponse.source;
+                return '';
+            }
+        }, {
+            key: 'pwtplt', //hb_format
+            val: function (bidResponse) {
+                // return bidResponse.mediaType;
+                return bidResponse.mediaType == "video" ? CONSTANTS.PLATFORM_VALUES.VIDEO : (bidResponse.native ? CONSTANTS.PLATFORM_VALUES.NATIVE : CONSTANTS.PLATFORM_VALUES.DISPLAY);
+            }
+        },
+        {
+        	key: 'pwtdid', // hb_deal
+        	val: function(bidResponse){ // todo: do we want to concat dealchannel as well?
+        		return bidResponse.dealId || '';
+        	}
+		},  
+		{
+        	key: 'pwtdeal', // hb_deal
+			val: function(bidResponse){ // todo: do we want to concat dealchannel as well?
+				if(bidResponse.dealId){
+					bidResponse.dealChannel = bidResponse.dealChannel || "PMP";
+					return bidResponse.dealChannel + CONSTANTS.COMMON.DEAL_KEY_VALUE_SEPARATOR + bidResponse.dealId + CONSTANTS.COMMON.DEAL_KEY_VALUE_SEPARATOR + bidResponse.adId;
 				}
+				return '';
+			}
+        },     
+        {
+            key: 'pwtbst', // our custom
+            val: function(bidResponse) {
+                return 1;
+            }
+        },               
+        {
+        	key: 'pwtpubid', // custom
+        	val: function(bidResponse){
+        		return CONFIG.getPublisherId();
+        	}
+        },
+        {
+        	key: 'pwtprofid', // custom
+        	val: function(bidResponse){
+        		return CONFIG.getProfileID();
+        	}
+        },
+        {
+        	key: 'pwtverid', // custom
+        	val: function(bidResponse){ // todo: empty value?
+        		return CONFIG.getProfileDisplayVersionID();
+        	}
+        },
+        {
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 9 tabs but found 8. (indent)](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15196)